tests/components/spreadsheet.test.ts

```diff
@@ -256,6 +256,16 @@ test("Spreadsheet detects frozen panes that exceed the limit size at start", asy
   expect(notifyUser).toHaveBeenCalled();
 });
 
+test("Warns user when viewport is too small for frozen panes but stops warning after resizing/Unmounted", async () => {
+  setDefaultSheetViewSize(0);
```

Comment:
I would add a comment here stating this is supposed to represent the "real life" where the default size is 0